src/main/java/me/EvsDev/EnderDragonTweaks/EnderDragonDeathListener.java

```diff
@@ -1,5 +1,7 @@
 package me.EvsDev.EnderDragonTweaks;
 
+import java.util.stream.Collectors;
+import static me.EvsDev.EnderDragonTweaks.ConfigManager.entry_playerDistanceFromOrigin;
```

Comment:
I don't really know how this managed to get here lol since so other config entry has it.

Reply:
not sure how it got there either...I think my IDE automatically added it and didn't remove it. I'll push a change to my branch to remove it.